middleware/index.ts

```diff
@@ -265,7 +254,7 @@ function applySfFxMiddleware(request: any, state: any, resultArgs: Array<any>):
     return [event, context, logger];
 }
 
-function getUserFn() {
+function getUserFn(): any {
```

Comment:
I wonder if we can do something like this would at least ensure we return what looks to be a function. It might also show us if we are making bad assumptions here (like returning undefined as default) that we could throw better error messages for.
```suggestion
function getUserFn(): (...args) : any {
```